style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --color-white: #fff;
+  --color-light: #7f8cff;
+  --color-light-dark: #6070ff;
+  --color-dark: #172b4d;
+  --color-dark-variant: #344563;
+  --transition: all 400ms ease;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background: var(--color-bg);
+  line-height: 1.7;
+}
+
+/* -------------------------------Styling the Nav Bar-------------------------------- */
+nav {
+  color: var(--color-light-dark);
+  font-weight: 700;
+  font-size: 18px;
+  padding: 10px 12px;
+  width: 100%;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav ul li {
+  margin: 4px 12px;
+}
+
+/* -- -------------------------------Styling the Home Page---------------------------------- -- */
+.main-page {
+  background-image: url('images/background-header.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
- [ ] :thinking: Kindly make sure the content is aligned with flexbox to meet the project requirements

- [ ] Kindly make sure there is a border bottom left radius for this section in order to stick to the design as stated in the project requirements :recycle: 

Your design:
![YourRadius](https://user-images.githubusercontent.com/97834160/212015095-98d5ecd9-0b23-4334-b97f-fb91813c34d2.png)


Figma design:
![FigmaRadius](https://user-images.githubusercontent.com/97834160/212015123-917ddddf-9bf6-455b-8cee-9c31922e0b20.png)